quick-start-nativescript/1-build-photo-album.md

```diff
@@ -69,22 +69,17 @@ The NativeScript companion app makes it easy for you to test your app on real de
 #### Action
 
 * **a.** Get your iOS or Android device.
-* **b.** Download and install the NativeScript companion app from the App Store or Google Play, respectively.
+* **b.** Download and install the Telerik Platform Companion app from your device's app store — i.e. the App Store for iOS users or Google Play for Android users (search for **Telerik Platform** or use the buttons provided below).
 
-[![iOS app store](images/app-store-icon.png)](https://itunes.apple.com/bg/app/nativescript/id882561588?mt=8)
-[![Google Play](images/google-play-icon.png)](https://play.google.com/store/apps/details?id=com.telerik.NativeScript&hl=en)
+[![iOS app store](images/app-store-icon.png)](https://itunes.apple.com/us/app/platform-companion/id1083895251?mt=8)
+[![Google Play](images/google-play-icon.png)](https://play.google.com/store/apps/details?id=com.telerik.PlatformCompanion&hl=en)
 
-* **c.** In the browser, select **Run** --> **Build**, select your platform (iOS/Android), choose **NativeScript companion app**, and click **Next**. You will see a QR code pointing to the application. 
-* **e.** **Android:** Open the NativeScript app on your device. With the NativeScript app running, open the notification center. Tap **Scan** and use the integrated scanner to scan the QR code displayed in the browser.
-![Using the notification center of your device](images/notification-center-buttons.png)
-* **f.** **iOS:** Open the NativeScript app on your device and with a two-finger left-to-right swipe from far left reveal the companion app menu. Tap **QR Scanner** in the menu and use the integrated scanner to scan the QR code displayed in the browser.
-![Using a two-finger swipe on your device](images/swipe.png)
-
-* **e.** **Universal Windows Platform:** A companion app for Windows Universal is not yet available.
+* **c**. Open the Telerik Platform Companion app on your device, and then select **Help & About**. In the newly opened page, locate the NativeScript section and select **Install Dev Application**. You will be redirected to your device's app store, where you should select **Install**. This will download and install the module required from Telerik Platform Companion app to run NativeScript applications on your device.
+* **d.** In the browser, select **Run** --> **Build**, select your platform (iOS/Android), choose **NativeScript companion app**, and click **Next**. Once the build finishes, select the **QR Scanner** option in the Companion app's menu and use the integrated scanner to scan the generated QR code.
```

Comment:
> This will download and install the module required from Telerik Platform Companion app to run NativeScript applications on your device.

I would be specific here with “install the module required”. Maybe “install the NativeScript player app, which the Telerik Platform companion app will use to run your applications.”

I think this will help set the expectation that the companion apps will switch apps when you get around to scanning the QR code.